.github/workflows/cypress-demo.yml

```diff
@@ -3,16 +3,11 @@ on:
   pull_request:
     types: [opened, synchronize]
   push:
-    branches:
-      - main
+    branches: [main]
 jobs:
-  cypress:
-    name: Cypress
+  e2e:
+    name: End-to-end integration tests
```

Comment:
We're going to have to update the required tests in the repository settings
![image](https://github.com/netlify/remix-compute/assets/3572990/c393cbeb-7abb-4d92-a5f6-50203063ebec)

Reply:
We should probably also rename the files `cypress-demo.yml` and `cypress-edge-demo.yml`

Reply:
Good catch!